demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementSpreadsheetImporter.php

```diff
@@ -176,15 +177,24 @@ public function process(SplFileInfo $workbook): void
             $originalStatementOrViolations = $builder->buildStatementAndReset();
```

Comment:
original Statement will be created nevertheless.
This will lead to OSTNs (including internId) in the DB without any way to access/delete these by the user.

Reply:
fix in [4e52b43](https://github.com/demos-europe/demosplan-core/pull/2575/commits/8586a664b1fd40f8f48c3106c1f375ee1eb30ef8)